### PR TITLE
`score-compose init|generate`

### PR DESCRIPTION
**Pull request by mathieu-benoit**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.score-compose/99-default.provisioners.yaml
```
Comment (astromechza) [inline .gitignore:1]: much better to ignore the entire directory right?

```
.score-compose/
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,42 @@
+score-compose:
```
Comment (astromechza) [inline Makefile:1]: If I can make a suggestion, a more canonical makefile would look like:

```
compose.yaml: score/score.yaml
	score-compose init --no-sample
	score-compose generate score/score.yaml \
		--build my-sample-container=app/ \
		--override-property containers.my-sample-container.variables.MESSAGE="Hello, Compose!"

.PHONY: compose-up
compose-up: compose.yaml
	docker compose up --build -d --remove-orphans

.PHONY: compose-test
compose-test: compose-up
	curl localhost:8080

.PHONY: compose-down
compose-down:
	docker compose down -v --remove-orphans || true
```

File: .gitignore (hunk 1 above)
Comment (mathieu-benoit) [reply .gitignore:1]: Good catch

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+compose-test:
+	curl localhost:8080
+
+compose-down:
+	docker compose down -v --remove-orphans
+
+score-helm:
+	score-helm run \
+		-f score.yaml \
```
[the remark this answers is not in the document]
Comment (mathieu-benoit) [reply Makefile:19]: Yup, fixed in other PR, thanks.

**Pull request merged**